workflows/sarek.nf

```diff
@@ -334,8 +323,26 @@ vep_genome         = params.vep_genome         ?: Channel.empty()
 vep_species        = params.vep_species        ?: Channel.empty()
 
 // Initialize files channels based on params, not defined within the params.genomes[params.genome] scope
-snpeff_cache       = params.snpeff_cache ? params.use_annotation_cache_keys ? Channel.fromPath("${params.snpeff_cache}/${params.snpeff_genome}.${params.snpeff_db}", checkIfExists: true).collect()   : Channel.fromPath(params.snpeff_cache).collect() : []
-vep_cache          = params.vep_cache    ? params.use_annotation_cache_keys ? Channel.fromPath("${params.vep_cache}/${params.vep_cache_version}_${params.vep_genome}", checkIfExists: true).collect() : Channel.fromPath(params.vep_cache).collect()    : []
+if (params.snpeff_cache && params.tools && params.tools.contains("snpeff")) {
+    def snpeff_annotation_cache_key = params.use_annotation_cache_keys ? "${params.snpeff_genome}.${params.snpeff_db}/" : ""
+    def snpeff_cache_dir =  "${snpeff_annotation_cache_key}${params.snpeff_genome}.${params.snpeff_db}"
```

Comment:
To double check: it is correct there is nothing between the `cache_key` and `snpeff_genome` variables, the string is meant to be straight after each other?

Reply:
that's it `*_annotation_cache_key` is either empty or a something like `genome.version/`